content/en/docs/azure/authentication-oidc.md

```diff
@@ -0,0 +1,212 @@
++++
+title = "Authentication using OIDC in Azure"
+description = "Authentication and authorization support through OIDC for Kubeflow in Azure"
+weight = 6
++++
+
+This section shows the how to setup Kubeflow with authentication and authorization support through OIDC in Azure using [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/).
+
+## Prerequisites
+
+- Install the [prerequisites for Kubeflow in Azure](/docs/azure/deploy/install-kubeflow)
+- [Register an application with the Microsoft Identity Platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#register-an-application)
+- [Add a client secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#add-a-client-secret)
+
+  **Note:**  Save your client Id, client secret, and tenant Id in a secure place to be used in the next steps to configure OIDC Auth Service.
+
+## Kubeflow configuration
+
+1. Download the latest kfctl {{% kf-latest-version %}} release from the [Kubeflow releases page](<https://github.com/kubeflow/kfctl/releases/tag/>{{% kf-latest-version %}}).
+
+1. Unpack the tar ball:
+
+    ```
+    tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
+    ```
+
+1. Run the below commands to build configuration files before deploying Kubeflow. The code below includes an optional command to add the binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
```

Comment:
There's still repetition here which stands out when reading the instructions: 

`...to add the binary kfctl to your path. If you don’t add the binary to your path,...`.

Maybe, as mentioned before, use:

`...to add the binary kfctl to your path - if you don’t do it,...`

WDYT?